frontend/src/views/process/ProcessList.vue

```diff
@@ -103,4 +107,15 @@
 <script setup>
 import SlideBar from "@/components/layouts/SlideBar.vue";
 import { ChevronUpIcon } from "@heroicons/vue/20/solid";
+import { computed, onMounted, ref } from "vue";
+import { useProcessStore } from "@/stores/process";
+
+const processStore = useProcessStore();
+const processes = ref([]);
+
+onMounted(async () => {
+    await processStore.fetchProcessesData();
+    processes.value = processStore.processes;
+});
+
```

Comment:
Delete extra line

Reply:
Addressed in [commit](https://github.com/carlos18bp/gym_project/pull/3/commits/7ac2f11757dc85d134b3c413bed38863f729857e)